ft_atoi.c

```diff
@@ -0,0 +1,53 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_atoi.c                                          :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ldamiens <ldamiens@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2021/11/06 15:43:54 by ldamiens          #+#    #+#             */
+/*   Updated: 2021/11/12 11:45:44 by ldamiens         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "libft.h"
+#include <stdio.h>
+
+int	ft_isspace(char character)
```

Comment:
Functions meant to be used in this file should be `static`
```suggestion
static int	ft_isspace(char character)
```